operations/finance.md

```diff
@@ -743,6 +759,156 @@ Removes selected bills. Bill must be empty, otherwise it's not possible to delet
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `BillId` | string | required | Unique identifier of the [Bill](finance.md#bill) to be closed. |
+| `Type` | string [Bill type](#bill-type) | required | Specifies the mode bill should be closed in. |
+| `BillCounterId` | string | optional | Unique identifier of the [Counter](enterprise.md#counter) to be used for closing. Default one is used when no value is provided. |
+| `FiscalMachineId` | string | optional | Unique identifier of the [Fiscal Machine](integrations.md#device) to be used for closing. Default one is used when no value is provided. |
+| `Options` | array of string [Bill options](#bill-options) | optional  | Options of the bill, only `DisplayCustomer` and `DisplayTaxation` can be used. If not provided both `DisplayCustomer` and `DisplayTaxation` are set. |
+| `TaxedDate` | string | optional | Date of consumption for tax purposes. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `DueDate` | string | optional | Deadline when [Bill](#bill) is due to be paid. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `VariableSymbol` | string | optional | Optional unique identifier of requested payment. Can be used only with [Bill type](#bill-type) `Invoice`. |
+| `TaxIdentifier` | string | optional | Tax identifier of account to be printed on a bill. |
+| `Notes` | string | optional | Notes to be attached to bill. |
+| `Address` | [Address parameters](customers.md#address-parameters) | optional | Address of the account to be displayed on bill. Overrides the default one taken from account profile. |
+
+### Response
+
+```javascript
+{
+    "Bills": [
+        {
+            "Id": "44eba542-193e-47c7-8077-abd7008eb206",
+            "CustomerId": "7eaf9da6-7229-454a-8cb0-abd700804bd2",
+            "CompanyId": null,
+            "CounterId": "84b25778-c1dd-48dc-8c00-ab3a00b6df14",
+            "State": "Closed",
+            "Type": "Receipt",
+            "Number": "84",
+            "VariableSymbol": null,
+            "CreatedUtc": "2020-06-11T08:39:32Z",
+            "IssuedUtc": "2020-06-25T08:49:38Z",
+            "TaxedUtc": "2020-06-25",
+            "PaidUtc": null,
+            "DueUtc": null,
+            "Notes": null,
+            "Options": [
+                "DisplayCustomer",
+                "DisplayTaxation"
+            ],
+            "Revenue": [],
+            "Payments": []
+        }
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Bills` | array of [Bill](#bill) | required | The closed bill. |
+
+## Get bill PDF
+
+Prints PDF version of closed bill. In case it's not possible to return PDF immediately returns unique event identifier which must be used in retried requests. 
```

Comment:
It actually doesn't print it. It creates a bill PDF - so I'd go with "Creates a PDF version of the specified bill. In case it's not possible to return PDF immediately, you must retry the call later while providing the unique event identifier that is returned from the first invocation."